manifests/params.pp

```diff
@@ -152,6 +152,27 @@
       }
     } # Windows
 
+    'FreeBSD': {
+      $conf_dir  = '/usr/local/etc/icinga2'
+      $log_dir   = '/var/log/icinga2'
+      $run_dir   = '/var/run/icinga2'
+      $spool_dir = '/var/spool/icinga2'
+      $cache_dir = '/var/cache/icinga2'
+      $pki_dir   = "${conf_dir}/pki"
+      $ca_dir    = '/var/icinga2/ca'
+      $user      = 'icinga'
+      $group     = 'icinga'
+      $lib_dir   = '/usr/local/lib/icinga2'
+      $constants = {
+        'PluginDir'          => "${lib_dir}/nagios/plugins",
+        'PluginContribDir'   => "${lib_dir}/nagios/plugins",
```

Comment:
You're right @lazyfrosch, that does look wrong.

What files are supposed to live in the `PluginContribDir`?  In the monitoring-plugins port, I see `/usr/local/libexec/nagios` where the `check_*` scripts live, and in the `icinga2` port I see `/usr/local/share/icinga2/include/plugins-contrib.d/` which contains a bunch of config files that define commands and such.  I'll adjust accordingly.